### PR TITLE
West Midlands_class 3- Mireille_Nyoubissi-Git/Github-week_No 2

### PR DESCRIPTION
**Pull request by MireilleNyoubissi**
I have made corrections to the answer to the questions.

# Your Details

- Your Name: Mireille Nyoubissi
- Your City: Birmingham
- Your Slack Name: Mireille Nyoubissi

# Homework Details

- Module: Git/Github
- Week: 2

-----
[View rendered homework.md](https://github.com/MireilleNyoubissi/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

**Comment by uzma000** [comment]
All the answers were correct but number 17, the visual studio spelling is incorrect and also the commit message means who and when the changes made. little bit add up.  but over all the answers were correct. great job

File: homework.md [hunk 1]
```diff
@@ -4,107 +4,110 @@
...
 
-Hyper Tall Massive Language
+Hypertext Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+It provides easy control over layout and presentation of webpages by separating content from design.
```
Comment (Haseeb3927) [inline homework.md:31]: Great response 👍

File: homework.md [hunk 2]
```diff
@@ -4,107 +4,110 @@
...
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language that enables us to program the behaviour of web pages.
```
Comment (Haseeb3927) [inline homework.md:46]: Well explained 💯

**Comment by Haseeb3927** [comment]
Well done Mireille. Your answers increased my knowledge as well 💯

File: homework.md (hunk 1 above)
Comment (MireilleNyoubissi) [reply homework.md:31]: Thanks, Haseeb

**Comment by MireilleNyoubissi** [comment]
> All the answers were correct but number 17, the visual studio spelling is incorrect and also the commit message means who and when the changes made. little bit add up. but over all the answers were correct. great job

Thank you for your input, Uzma.

**Pull request closed without merging**